src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -45,7 +45,7 @@ void HexagonGame::initLua_Utils()
     lua.writeVariable("u_isKeyPressed",
         [this](int mKey) { return window.getInputState()[KKey(mKey)]; });
     lua.writeVariable(
-        "u_haltTime", [this](float mDuration) { status.timeStop = mDuration; });
+        "u_haltTime", [this](float mDuration) { status.pauseTime(mDuration); });
```

Comment:
I think you are pausing this in seconds notation. This is supposed to be in frames. Just quickly divide this by 60 and you're good.